webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -1005,26 +1005,31 @@ sealed trait QueryTrait {
     override def hashCode() = returnType.hashCode * 17 + composition.hashCode * 3 + transform.hashCode * 11 + criteria.toSet.hashCode * 7
 }
 
-final case class Query(
+final case class OldQuery(
```

Comment:
this renaming will be binary incompatible no? It will break backs the other plugin.

Reply:
It won't because we have an apply ... I think it still could break if there was some pattern patching though due to different unapply method

Reply:
But in the previous case, an apply in the object wasn't enought to keep binary compat. Why it is, here?